src/server/db_slice.cc

```diff
@@ -612,22 +616,11 @@ void DbSlice::FlushDb(DbIndex db_ind) {
     CreateDb(db_ind);
     db_arr_[db_ind]->trans_locks.swap(db_ptr->trans_locks);
 
-    auto cb = [this, db_ptr = std::move(db_ptr)]() mutable {
```

Comment:
because we dont need now to call PerformDeletion from flush db I removed this from running on fiber